Source_Code/TM_Backend/TeamMentor.Rest/Controllers/TM_REST_Admin.cs

```diff
@@ -147,6 +149,47 @@ public Stream TBot_List()
             UserRole.Admin.demand();
             return TmWebServices.XmlDatabase_ReloadData();	        
         }
+
+        [Admin]
+        public bool Publish_Data()
+        {
+            UserRole.Admin.demand();
+            if (TMConfig.Current.notNull())
+            {
+                var scriptPath = TMConfig.Current.TMSetup.TmReloadDataScriptPath;
+                if (!scriptPath.fileExists())
+                {
+                    "[Publishing Data]You need to set the location of the bat file".log();
+                    return false;
+                }
+                var processInfo = new ProcessStartInfo()
+                {
+                    FileName                = scriptPath, 
+                    CreateNoWindow          = true,
+                    UseShellExecute         = false,
+                    RedirectStandardError   = true,
+                    RedirectStandardOutput  = true,
+                    WorkingDirectory        = Path.GetDirectoryName(scriptPath)
+                };
+
+                using (var process = Process.Start(processInfo))
+                {
+                    if (process == null) return false;
+                    process.Start();
+                    process.WaitForExit();
+                    process.StandardOutput.ReadToEnd().log();
+                    if (process.StandardError.ReadToEnd().notNull() && process.StandardError.ReadToEnd().Length > 0)
+                    {
+                        "[Publishing Data]Error publishing data ".log();
+                        process.StandardError.ReadToEnd().log();
+                        return false;
+                    }

```

Comment:
can you replace this with the FluentSharp process method? (easier to read). And open an issue on FluentSharp to add the ```using``` to it